docs/prompting/zero_shot/re2.md

```diff
@@ -27,7 +27,7 @@ client = instructor.from_openai(OpenAI())
 
 
 class Solution(BaseModel):
-    final_answer: int = Field(..., description="This is the final answer")
+    final_answer: int = Field(..., description="Final Answer")
```

Comment:
why would we need the description final answer if the variable name is final answer